falcon/testing/client.py

```diff
@@ -134,8 +135,20 @@ def simulate_request(app, method='GET', path='/', query_string=None,
             method (str): An HTTP method to use in the request
                 (default: 'GET')
             path (str): The URL path to request (default: '/')
+            params (dict): A dictionary of query string parameters,
+                where each key is a parameter name, and each value is
+                either a ``str`` or something that can be converted
+                into a ``str``, or a list of such values. If a ``list``,
+                the value will be converted to a comma-delimited string
+                of values (e.g., 'thing=1,2,3').
+            params_csv (bool): Set to ``False`` to encode list values
+                in query string params by specifying multiple instances
+                of the parameter (e.g., 'thing=1&thing=2&thing=3').
+                Otherwise, parameters will be encoded as comma-separated
+                values (e.g., 'thing=1,2,3'). Defaults to ``True``.
```

Comment:
At some point we may want to go throughout the framework and switch the defaults to use form-based encoding instead of csv encoding. Keeping the default to `True` for now for the sake of consistency.